src/components/TableV3/TablePager.tsx

```diff
@@ -0,0 +1,58 @@
+import { Button } from '@/components/ButtonNew/Button'
+import { Paragraph } from '@/components/TypographyNew'
+import React, { useState } from 'react'
+
+interface TablePagerProps {
+  pageSize: number
+  totalItems: number
+  pagedItemName: string
+  onPageChange: (startIndex: number, endIndex: number) => void
+}
+
+export const TablePager: React.FC<TablePagerProps> = ({
+  pageSize,
+  totalItems,
+  pagedItemName,
+  onPageChange,
+}) => {
+  const [currentPage, setCurrentPage] = useState(0)
+  let start = totalItems === 0 ? 0 : currentPage * pageSize + 1
```

Comment:
perhaps we could check for `totalItems` first and if 0-sized only render the empty `PagerContainer`?